build.sh

```diff
@@ -55,10 +55,6 @@ while getopts ":p" opt; do
     case $opt in
         p)
         JEKYLL_CMD="serve --baseurl= --watch --trace --incremental"
-        ;;
-        i)
-        [[ `${RUBY} -v` =~ 'ruby 1' ]] && echo "Error: building the docs with the incremental option requires at least ruby 2.0" && exit 1
-        JEKYLL_CMD="liveserve --baseurl= --watch --incremental"
     esac
```

Comment:
What's wrong with the incremental build?

Reply:
two things:

1. it is not activated because it is missing in the `getopts` line [above](https://github.com/apache/flink-web/blob/asf-site/build.sh#L54) and none of these case statements will be used
2. if activated (instead of removing it), this will lead to the following error:
```
bundle exec jekyll liveserve --baseurl= --watch --incremental --source /home/nico/Projects/flink-web --destination /home/nico/Projects/flink-web/content
jekyll 3.0.5 | Error:  Whoops, we can't understand your command.
jekyll 3.0.5 | Error:  invalid option: --baseurl=
jekyll 3.0.5 | Error:  Run your command again with the --help switch to see available options.
```

Reply:
Okay, thanks.